src/server/src/main/java/io/cassandrareaper/ReaperApplication.java

```diff
@@ -179,6 +179,10 @@ public void run(ReaperApplicationConfiguration config, Environment environment)
         repairThreads, maxParallelRepairs);
 
     tryInitializeStorage(config, environment);
+    if (config.getSchemaMigrationMode()) {
+      LOG.info("Reaper is running in schema migration mode. Exiting now that the schema was created.");
```

Comment:
And if this fails, the normal stack trace is just thrown? Does it indicate enough to the user that the initialisation only failed? Or could we at least in the startup say "running in schema migration only" instead of only at the successful exit?